ui/components/subscription/TeamListing.tsx

```diff
@@ -23,22 +22,20 @@ type TeamListingProps = {
   selectedChain: any
   listing: TeamListing
   teamContract: any
-  teamSplitAddress: string | undefined
   marketplaceTableContract?: any
   refreshListings?: any
   editable?: boolean
-  teamName?: string
+  teamName?: boolean
```

Comment:
Why boolean here?

Reply:
I was passing the teamName to that component, now I'm fetching the teamName in the TeamListing component so I changed it to a boolean.